CHANGELOG.md

```diff
@@ -42,9 +42,10 @@ Mapbox welcomes participation and contributions from everyone.
 * Fix location indicator layer rendering when SwiftShader is used. ([#1105](https://github.com/mapbox/mapbox-maps-android/pull/1105))
 * Avoid possible crash at program exit caused by dummy tracer accessed after destruction. ([#1116](https://github.com/mapbox/mapbox-maps-android/pull/1116))
 * Fix crash for the case when a map event is handled by an Observer of a destructed map. ([#1116](https://github.com/mapbox/mapbox-maps-android/pull/1116))
+* Deliver event to observable subscribers even if observable is already destructed. ([#1134](https://github.com/mapbox/mapbox-maps-android/pull/1134))
```

Comment:
IMO we don't need that entry as this was working fine before #1116 and we actually return to previous correct behaviour.

Reply:
Good point, should we remove the entries from #1116 too, as this is reverted now?

Reply:
removed this entry.